clusterloader2/pkg/measurement/common/scheduling_throughput.go

```diff
@@ -82,7 +82,15 @@ func (s *schedulingThroughputMeasurement) Execute(config *measurement.Config) ([
 		if err != nil {
 			klog.Warningf("error while getting threshold param: %v", err)
 		}
-		return s.gather(threshold)
+		enableViolations, err := util.GetBoolOrDefault(config.Params, "enableViolations", true)
+		if err != nil {
+			klog.Warningf("error while getting enableViolations param: %v", err)
+		}
+		summary, err := s.gather(threshold)
+		if err != nil && errors.IsMetricViolationError(err) && !enableViolations {
```

Comment:
Can you please make it consistent with what is here:
https://github.com/kubernetes/perf-tests/blob/4d71498446f84fd2b371e1506748fb2a986c0d7f/clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go#L89